### PR TITLE
Fixed initial vizualization in pure Python

### PR DESCRIPTION
**Pull request by tjsego**

### CODE REVIEW

File: cc3d/core/GraphicsUtils/CC3DPyGraphicsFrame.py [hunk 1]
```diff
@@ -348,6 +348,9 @@ def __init__(self,
     def start(self, auto_shutdown: bool = True):
         """Start the event loop"""
 
+        self.draw()
```
Comment (maciekswat) [inline cc3d/core/GraphicsUtils/CC3DPyGraphicsFrame.py:351]: does this work fine in the PLayer as well?

Comment (tjsego) [reply cc3d/core/GraphicsUtils/CC3DPyGraphicsFrame.py:351]: No effect in player. The player implementation doesn't derive from this class.

Comment (maciekswat) [reply cc3d/core/GraphicsUtils/CC3DPyGraphicsFrame.py:351]: perfect!

**Review by maciekswat — APPROVED** [review]
approved

**Pull request merged**